crates/re_analytics/Cargo.toml

```diff
@@ -12,6 +12,9 @@ repository.workspace = true
 rust-version.workspace = true
 version.workspace = true
 
+[lints]
+workspace = true
```

Comment:
This is a bit annoying that we have to remember… maybe we should add a lint for this at some point